t/plugin/response-rewrite.t

```diff
@@ -509,3 +509,96 @@ GET /t
 additional properties forbidden, found invalid_att
 --- no_error_log
 [error]
+
+
+
+=== TEST 17: add validate vars
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.response-rewrite")
+            local ok, err = plugin.check_schema({
+                vars = {
+                    {"status","==",200}
```

Comment:
Need a case fails to validate, as the successful case is already covered later.

Reply:
added